code/modules/chemistry/Chemistry-Holder.dm

```diff
@@ -351,7 +351,6 @@ proc/chem_helmet_check(mob/living/carbon/human/H, var/what_liquid="hot")
 					target_reagents.add_reagent(reagent_id, receive_amt, current_reagent.data, src.total_temperature, TRUE, TRUE)
 					current_reagent.on_transfer(src, target_reagents, receive_amt)
 					src.remove_reagent(reagent_id, transfer_amt, FALSE, FALSE)
-					return 0
```

Comment:
Shouldn't this `break` the loop instead then? (Also possibly for the `return 0` above as well?)